github/github.go

```diff
@@ -880,6 +880,23 @@ func cloneRequest(r *http.Request) *http.Request {
 	return r2
 }
 
+func formatDuration(d time.Duration) string {
```

Comment:
Sounds good, thanks.

> Is it possible to assign the issue to me?

@gmlewis would have to invite you to org, otherwise we can't do it because of weird GitHub limitations (/cc @kytrinyx FYI, this isn't very friendly to open source projects):

![image](https://user-images.githubusercontent.com/1924134/27744602-23c780d8-5d8e-11e7-9f84-65b0bfc78794.png)

I'll just post a comment in the issue to point to this PR, so people will be aware that you're working on it here.